source/deployment/push.rst

```diff
@@ -150,31 +150,39 @@ Performance Benchmarks for Mobile Applications
 
 Properly configured mobile applications on 4G/LTE or Wifi should perform as follows: 
 
-iPhone Plus 6s on 4G/LTE connection (50 ms ping time, 50 Mbps upload, 8 Mbps download): 
+iPhone 6s Plus on 4G/LTE connection (50 ms ping time, 50 Mb/s download, 8 Mb/s upload): 
 
-- **Loading a new channel:** less than 4 seconds 
+- **Loading a new channel:** less than 4 seconds
 - **Returning to a channel previously viewed:** less than 1 second
-- **Switching back to app after it's recently been in the background:** less than 1 second 
-- **Switching to app and loading channel after phone has been asleep:** less than 5 seconds 
-- **Cold start of app until first page load:** 10 seconds 
+- **Switching to the app after previously opening it:** less than 1 second
```

Comment:
Hey @DavidLu1997, I'm not sure that this is very clear. I'd propose the original wording is probably okay?